CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Config: Open vertical splits from floating windows with `tools.float_win_config.open_split_vertical`.
```

Comment:
I'm on my mobile, so I can't tell for sure right now, but I think markdownlint will fail here. 

```suggestion
- Config: Open vertical splits from floating windows with 
  `tools.float_win_config.open_split_vertical`.
```

Reply:
Looks like it passed.